### PR TITLE
adding restart build script

### PR DESCRIPTION
**Pull request by dennisnewel**

### CODE REVIEW

File: utilities/codeship_restart_build.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/home/rof/.rvm/rubies/ruby-2.2.5/bin/ruby
```
Comment (mlocher) [inline utilities/codeship_restart_build.sh]: I would replace that with `#!/usr/bin/env ruby`, that way that script works outside our build VMs as well.

**Review by mlocher — CHANGES_REQUESTED** [review]
One minor issue to make this work outside of Codeship build VMs as well.

File: utilities/codeship_restart_build.sh (hunk 1 above)
Comment (dennisnewel) [reply utilities/codeship_restart_build.sh]: Doh! i'll change that

**Review by mlocher — APPROVED** [review]
:shipit:

**Pull request merged**